docs/table_design/Data_distribution.md

```diff
@@ -112,51 +214,208 @@ CREATE TABLE site_access(
     pv BIGINT SUM DEFAULT '0'
 )
 AGGREGATE KEY(event_day, site_id, city_code, user_name)
--- Set the partitioning method as Range partitioning.
-PARTITION BY RANGE(event_day) (
+PARTITION BY RANGE(event_day)(
     PARTITION p1 VALUES LESS THAN ("2020-01-31"),
     PARTITION p2 VALUES LESS THAN ("2020-02-29"),
     PARTITION p3 VALUES LESS THAN ("2020-03-31")
 )
--- Set the bucketing method as Hash bucketing and specify the bucketing key.
-DISTRIBUTED BY HASH(event_day, site_id);
+DISTRIBUTED BY HASH(site_id);
+```
+
+**Create multiple partitions at a time**
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/f9258a22-80de-451a-8f59-7f59ce1972ac)
In a previous table, you have defined this as "Create partitions in batch"